components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1657,6 +1662,47 @@ RewardsNotificationService* RewardsServiceImpl::GetNotificationService() const {
   return notification_service_.get();
 }
 
+void RewardsServiceImpl::StartNotificationTimer() {
+  notification_timer_ = std::make_unique<base::RepeatingTimer>();
+  notification_timer_->Start(FROM_HERE, base::TimeDelta::FromMinutes(15), this,
```

Comment:
I think we should check it on startup and then every few hours or once per day. I think 15min interval is not needed for this one

Reply:
OK, that makes sense.